js/repeater.js

```diff
@@ -211,6 +211,46 @@
 			return markup;
 		},
 
+		disable: function() {
+			var disable = 'disable';
+			var disabled = 'disabled';
+
+			this.$search.search(disable);
+			this.$filters.selectlist(disable);
+			this.$views.find('label').attr(disabled, disabled);
+			this.$pageSize.selectlist(disable);
+			this.$primaryPaging.find('.combobox').combobox(disable);
+			this.$secondaryPaging.attr(disabled, disabled);
+			this.$prevBtn.attr(disabled, disabled);
+			this.$nextBtn.attr(disabled, disabled);
+
+			this.$element.addClass('disabled');
+			this.$element.trigger('disabled.fu.repeater');
+		},
+
+		enable: function() {
+			var disabled = 'disabled';
+			var enable = 'enable';
+			var pageEnd = 'page-end';
+
+			this.$search.search(enable);
+			this.$filters.selectlist(enable);
+			this.$views.find('label').removeAttr(disabled);
+			this.$pageSize.selectlist('enable');
+			this.$primaryPaging.find('.combobox').combobox(enable);
+			this.$secondaryPaging.removeAttr(disabled);
+
+			if(!this.$prevBtn.hasClass(pageEnd)){
+				this.$prevBtn.removeAttr(disabled);
```

Comment:
this.$prevBtn.prop(disabled, false);

Do _not_ use removeProp() (see https://api.jquery.com/removeProp/#entry-longdesc)